src/profile_service/index.js

```diff
@@ -21,6 +21,7 @@ ActivityDetails.prototype = {
   getUserId() { return this.receipt.remember_me_id; },
   getUserProfile() { return this.profile; },
   getOutcome() { return this.receipt.sharing_outcome; },
+  getBase64SelfieUri() { return this.profile.base64SelfieUri; },
 };
 
 const payload = new Payload('');
```

Comment:
Unrelated to this commit - is this right `console.log(`Error getting response data: $${err.message}`);` - 2 `$$`?

Reply:
Yup, I'll change it 👍